tests/route/test_route_flow_counter.py

```diff
@@ -214,9 +214,4 @@ def _get_nexthop(self, duthost, ipv6):
         else:
             cmd = 'show ip bgp summary'
         parse_result = duthost.show_and_parse(cmd)
```

Comment:
just want to catch up on this.
after merging the revert PR, what is the change for output of `show ip bgp sum` that we need to to remove below lines?

Reply:
nvm, I understand now, it's fixing a typo in `neighbor`